lib/utils.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+create_hsm_config() {
+    # Create a configuration file
+    cat > $TMPDIR/softhsm.conf << EOF
+directories.tokendir = $TMPDIR
+objectstore.backend = file
+log.level = DEBUG
+EOF
+}
+
+install_softhsm() {
+    # TODO: the test may only run on latest Feodra and RHEL>9 versions:
+    #   https://issues.redhat.com/browse/RHEL-34856
+    yum install softhsm -y
```

Comment:
Why not to use `dnf` here then? Both RHEL9 and Fedora use `dnf`.

Reply:
And is there a reason not to wrap these calls with `rlRun`? If they fail, it wont be reported anywhere.

Reply:
I added both the `dnf` and the `rlRun`s to wrap those calls. Thanks for the hint!